### DISCUSSION TITLE
Task option to install/uninstall mods

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#705] [Suggestions]
Hey Max,

As we can install and uninstall mods while a server is running for ASA, would it be possible to have a task that can do this also?

For example, during weekends additional mods may be installed to offer different gameplay to a server. 
It would allow for server events where during a certain timeframe event type mods are installed and uninstalled after.

Thank you,
Gem

**Comment by mvaessen** [comment]
Think that should be possible 👍

**Comment by mvaessen** [comment]
closing as this is a duplicate
